package.json

```diff
@@ -23,7 +23,7 @@
 		"postpack": "npm run clean",
 		"build": "rspack build",
 		"typecheck": "tsc --noEmit",
-		"test": "node --test ./test"
+		"test": "node --test --experimental-test-snapshots --test-update-snapshots --require ./test/snapshot-config.js ./test/index.test.js"
```

Comment:
the creation of snapshots should be run in a different command like
`test:regenerate`